src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/CourseVersionError.java

```diff
@@ -0,0 +1,21 @@
+package fi.aalto.cs.apluscourses.intellij.notifications;
+
+import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;
+
+public class CourseVersionError extends Notification {
+  /**
+   * Notification to be shown when the plugin is outdated with respect to the current course.
+   *
+   * @param isError If true, the notification is an error. Otherwise, it is a warning.
+   */
+  public CourseVersionError(boolean isError) {
```

Comment:
Nit: perhaps polymorphism, that is, separate classes for warning and error, would make more sense than differentiating between them by a boolean variable. After all, they have very little in common.